.github/workflows/automatic_pull_request_review.yml

```diff
@@ -0,0 +1,82 @@
+name: Automatic pull request review
```

Comment:
It seems everything is good to go, @minhnimble should we update to add this workflow to our `cicdtemplate` too? Same to what you did with Bitrise?

![image](https://user-images.githubusercontent.com/16315358/135995735-267e9118-8c1e-4229-8954-69c731a5b26e.png)

Reply:
ohh I am not aware we have the templates stored in `cicdtemplate`, sure let me add it 👍

Reply:
@minhnimble I meant same to what you did in this PR https://github.com/nimblehq/android-templates/pull/130 🤓